### ISSUE TITLE
config.ini file completely gone missing

### ISSUE THREAD

**Issue by atharvak95** [owasp-amass/amass#955]
config.ini needs to be brought back from the earlier version. the config.yaml file in examples folder is pathetic, does not even ask for API keys anymore. There's a file called resources.yaml which does ask but there is no instructions on what to do with it. This is the most pathetic update of amass ever.

**Comment by soufianetahiri** [comment]
Stay cool dude don't forget that this is a free tool maintained for free by some awesome people.

**Comment by sa3hin** [comment]
> config.ini needs to be brought back from the earlier version. the config.yaml file in examples folder is pathetic, does not even ask for API keys anymore. There's a file called resources.yaml which does ask but there is no instructions on what to do with it. This is the most pathetic update of amass ever.

You can use old version
`go install -v github.com/owasp-amass/amass/v3/...@master`

**Comment by yosignals** [comment]
I came to see what had changed with amass, 106 open issues around config file I didnt want to make it 107, or be kettled into another platform to seek answers, @sa3hin thanks for the tip! 

using the old version isnt ideal, Amass had been kind to me in the past, it's a shame that to many people are falling over the same issue that isnt getting any focus

**Comment by caffix** [comment]
Many people are not reading the instructions. The only version of Amass that is active and up-to-date right now is the Docker Compose version. There is no longer a `config.ini` file. The `config.yaml` and `datasources.yaml` files make up the configuration settings you were previously used to.

**Comment by yosignals** [comment]
Yeah the docker instance was just as frustrating, anyway, no bother, I’ve just moved to subfinder, but I used to love amass, seems now quite amess especially if so many people are failing to use it #UX Such is life 😊Sent from my iPhoneOn 1 Mar 2025, at 18:22, Jeff Foley ***@***.***> wrote:﻿
  Many people are not reading the instructions. The only version of Amass that is active and up-to-date right now is the Docker Compose version. There is no longer a config.ini file. The config.yaml and datasources.yaml files make up the configuration settings you were previously used to.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>



caffix left a comment (owasp-amass/amass#955)
Many people are not reading the instructions. The only version of Amass that is active and up-to-date right now is the Docker Compose version. There is no longer a config.ini file. The config.yaml and datasources.yaml files make up the configuration settings you were previously used to.

**Comment by caffix** [comment]
I'm sure that Subfinder will be enough for many people. Take care
